pydantic/fields.py

```diff
@@ -32,11 +32,21 @@ class Validator(NamedTuple):
     check_fields: bool
 
 
+class Schema:
```

Comment:
Adding a docstring and/or comment would be helpful here to know what this class is for.

Reply:
will do, I need to do docs for this in general.